yucca/run/run_inference.py

```diff
@@ -111,6 +125,21 @@ def main():
         required=False,
         help="Save softmax outputs. Required for softmax fusion.",
     )
+    parser.add_argument(
+        "--batch_size",
+        "-bs",
+        default=16,
+        type=int,
+        required=False,
+        help="Batch size for inference. Defaults to 16.",
+    )
```

Comment:
ATM Inference is only supported for batches of 1 because the WritePredictions is implemented on_batch_end and just writes whatever is predicted there. 

We can either implement that to write predictions per case in a batch or we can remove the posibility of setting batch_size>1 during inference.

Reply:
fixing it to 1 for now.